src/Generator.h

```diff
@@ -1363,6 +1363,18 @@ class GeneratorInput_Buffer : public GeneratorInputImpl<T, Func>, public Constra
         this->estimate_impl(var, min, extent);
         return *this;
     }
+
+    Func in() {
```

Comment:
Shouldn't this just go in a trivial PR or submit to master? @steven-johnson

Reply:
Wasn't sure what we've decided w.r.t forwarding methods from ```Input<Func>``` I'll wait to see what Steven says.